sftp_lambda_integration/pyproject.toml

```diff
@@ -5,11 +5,9 @@ description = "Code related to Lambda trigger and SFTP upload integration"
 package-mode = false
 
 [tool.poetry.dependencies]
-python = "~3.9"
-pandas = "^1.4.2"
-gen3 = "^4.11.4"
```

Comment:
i thought we needed pandas and gen3 in `extractData.py`?

Reply:
Ahh! That is an oversight from my side. My focus was so narrowed on to the sftp script, I missed looking beyond it. Thank you Pauline.